exercises/03_count_byte/main_test.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestCountByte(t *testing.T) {
+	for _, tc := range []struct {
+		name      string
+		input     string
+		inputbyte byte
+		want      int
+	}{
+		{"empty", "", 101, 0},
```

Comment:
jfyi: Do you remember what is ascii 101? I recommend using 'e':
```go
{"empty", "", 'e', 0},
```
Alternatively, you could use just zeroes here:
```go
{"empty", "", 0, 0},
```